### ISSUE TITLE
The game crashed whilst ticking block entity

### ISSUE THREAD

**Issue by Pawfulness** [EnigmaticaModpacks/Enigmatica6#4410]
### Modpack Version

1.0.0 Expert

### Describe your issue.

Game crashed while I was falling in to a cave. 
I'm not able to load the world anymore.

### Crash Report

https://gist.github.com/Pawfulness/43f78e220dfc69f2e20da1421529a42c

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by Pawfulness** [comment]
No, nothing like that. 
Let me know if you need anything.

**Comment by NillerMedDild** [comment]
@MaxNeedsSnacks got a RoadRunner issue here with a very familiar looking stacktrace, do you recall how this is resolved / whether it was ever fixed?

**Comment by MaxNeedsSnacks** [comment]
Nope, we never got to the root cause of it unfortunately since it was basically impossible for us to reliably reproduce.

@Pawfulness Do you mind sending me your world download over Discord or something perhaps? Maybe we can finally gain some insight into this issue. (Also, to solve this, remove RoadRunner, load into the world, and readd it iirc)

**Comment by Pawfulness** [comment]
Sorry, I only have a backup save of the world, a day before the issue occurred. I can send it over, if needed. 
Discord: Pawless#1747

I fixed the issue with NBT Explorer, by removing the block entity at the coordinates, in the crash report.
This removed the 2 water bees and the honey level (5) in the vanilla beehive. 
The chunk was also claimed and force loaded.

**Comment by Pawfulness** [comment]
I will try to reproduce it and get a save file.

**Comment by Pawfulness** [comment]
I got it again. https://gist.github.com/Pawfulness/f8f998e4351446205f5dcb43d2b3ecbc 
Forgot to mention that, there were 3 water bees assigned to the beehive. The third one was bred. 
Maybe the bee growing up inside the hive caused the issue.
